blacksmith/tools/templates/pytorch_template.py

```diff
@@ -0,0 +1,159 @@
+# SPDX-FileCopyrightText: (c) 2025 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+import os
+import traceback
+
+import torch
+from tqdm import tqdm
```

Comment:
nit: Could we keep imports grouped for template:
```suggestion
import os
import traceback
import torch

from tqdm import tqdm
```